### DISCUSSION TITLE
Is it possible to pick a printer profile when starting the GUI

### DISCUSSION THREAD

**Discussion by stefanfoulis** [supermerill/SuperSlicer#2674] [Q&A] [answered]
For my workflow it would be useful if I could create multiple shortcuts to start SuperSlicer, each loading an instance of the application with a different printer profile. So I could slice for the different printers side-by-side.

(I'm on linux using the AppImage)

**Comment by neophyl** [comment, marked as answer]
The usual thing that many of us do is to take advantage of the way Project files work.  A project contains not only the models you load but also all the settings you have currently selected.  So for example if I save a Default_Mk3.3mf with my prusa printer and usual filament and print setting I want and then a Default_cr10.3mf for my creality I can start the program by opening which ever project I want.  
This works as long as SuSi is the default program for a double click.  If not then its just as easy to start SuSi and then open the project.  

I've not tried but its also probably possible to make a shortcut with a project as the target so you can do the same with a shortcut, at least in windows, but Im not a Linux user so you would almost certainly know more about that.
